### ISSUE TITLE
Provide an example of non-CLI environment

### ISSUE THREAD

**Issue by AlexElin** [angular/angular#48517]
### Describe the problem that you experienced

The [docs](https://angular.io/guide/testing-components-scenarios#calling-compilecomponents) contain the sentence:
> If you run tests in a non-CLI environment, the tests might fail with a message like this one:

I think it would be helpful to provide 1-2 examples of non-cli environemnts. So the sentence will look like this:
"If you run tests in a non-CLI environment (e,g, {example_of_the_such_environment}), the tests might fail with a message like this one:"

### Enter the URL of the topic with the problem

https://angular.io/guide/testing-components-scenarios#calling-compilecomponents

**Comment by slim-codes** [comment]
I agree with. That sentence makes you wonder what a **non-CLI environment** is.

**Comment by bob-watson** [comment]
I added **state: needs eng input** to clarify the intention and context of this statement.

> If you run tests in a non-CLI environment, the tests might fail with a message like this one:

**Comment by JeanMeche** [comment]
The [page you linked](https://angular.io/guide/testing-components-scenarios#calling-compilecomponents) is outdated.

The error message was droped with v13. I could help update it, if only I could recreate an example where `compileComponents()` is required but I wasn't able to reproduce it yet.
